packaging/archlinux/PKGBUILD

```diff
@@ -1,5 +1,30 @@
 pkgname=librdkafka
-pkgver=master
+pkgver=1.0.0.RC5.r11.g3cf68480
 pkgrel=1
-pkgdesc=The Apache Kafka C/C++ client library
-arch=('i686' 'x86_64')
+pkgdesc='The Apache Kafka C/C++ client library'
+url='https://github.com/edenhill/librdkafka'
+license=('BSD')
+arch=('x86_64')
```

Comment:
what about 32bit? (i686?)

Reply:
Isn't supported by Arch anymore, i686 has been dropped: https://www.archlinux.org/news/the-end-of-i686-support/